docs/AI.md

```diff
@@ -0,0 +1,65 @@
+<!--
+Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+
+SPDX-License-Identifier: curl
+-->
+
+# On AI use in curl
+
+These are guidelines for AI use when contributing to curl.
+
+## For security reports and other issues
+
+If you asked an AI tool to find problems in curl, you **must** make sure to
+reveal this fact in your report because we have little faith in AI's ability
+to do this accurately.
```

Comment:
Suggestion: “because we have little faith in” could trigger a “well, _I_ can and will prove them otherwise” response, the opposite of what is intended. As an alternative I’d suggest something that sounds more matter-of-fact-ly, along the lines of:

> [...] reveal this fact in your report. This hard requirement exists due to the huge amount of hallucinations and inaccuracies in AI-involved bug reports that were sent to the curl project in the past.

Reply:
Fair point, thanks. I'll rephrase.

Reply:
Actually, it is also mention in the paragraph below, so I can probably just shorten the first one. Check the update!

Reply:
Yes, good point! I think that works, too.